### DISCUSSION TITLE
Inconsistent snapping with multiple monitors?

### DISCUSSION THREAD

**Discussion by deitrix** [rxhanson/RectanglePro-Community#231] [Q&A] [answered]
I'm using multiple monitors with Rectangle Pro. I'm using both the "Edges & Corners" function, as well as "Snap Targets" configured for a set of Custom Shortcuts. I've got a keyboard shortcut set up to only show snap targets when the Shift key is pressed. My understanding is that this only applies to snap targets, and not the Edges & Corners.

On my primary monitor, the "Edges & Corners" snap targets show when I'm not pressing Shift. All I have to do is drag the window to the edge of the screen to trigger it. Whereas, on my secondary monitor, they don't show at all unless I'm holding Shift, regardless of where I position the window.

Is there some option I'm missing?

![image](https://user-images.githubusercontent.com/14089691/220583551-f1f143ab-9add-41a8-97b7-d8da752bf3a1.png)

![image](https://user-images.githubusercontent.com/14089691/220583742-4795ce6d-fb2e-4dc5-b208-4e95173895e4.png)

**Comment by rxhanson** [comment, marked as answer]
Thanks for reporting. It's a bug, and I have a fix in place that will go into the next release. I'll let you know if I create a beta release sooner.

**Reply by deitrix** [reply]
Brilliant. Thank you!
